ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -1001,8 +1042,6 @@ public class BottomSheetController: UIViewController, Shadowable {
         // Minimum padding from top when the sheet is fully expanded
         static let minimumTopExpandedPadding: CGFloat = 25.0
 
-        static let cornerRadius: CGFloat = 14
-
         static let expandedContentAlphaTransitionLength: CGFloat = 30
 
         static let maxSheetWidth: CGFloat = 610
```

Comment:
why isn't maxSheetWidth not part of tokens? similar comment for other constants in this file?

Reply:
They aren't listed out as tokens in the figma: https://www.figma.com/file/b7dGSxeToclUNKhwgN2KL5/Sheets-(mobile)?node-id=1%3A15301

Reply:
right because spacing token isn't a public token we let clients to change. but simialr to your tooltip and other tokenize control, we have move these type of private constant still in the tokenset file (as private extension) to define them.

Reply:
@sophialee0416  / @huwilkes  : am I incorrect?

Reply:
fwiw, i just looked through other tokenized controls and tooltip, segmented control, and HUD all have constants still
tooltip:
<img width="397" alt="image" src="https://user-images.githubusercontent.com/31874971/217440026-c1e8a554-8209-4732-8a3c-675fc03bdc8c.png">

HUD:
<img width="408" alt="image" src="https://user-images.githubusercontent.com/31874971/217440084-912c9974-e1fc-44a3-80bc-8e80f47a5765.png">

Segmented Control:
<img width="408" alt="image" src="https://user-images.githubusercontent.com/31874971/217440152-554a26b5-6743-4cbc-ace3-b39d5b35a16c.png">

Reply:
I think we generally only move things that were in the designs in to the tokens. Some of these might be worth following up with design about adding to the tokens, though.